tools/golang/Dockerfile

```diff
@@ -15,24 +15,20 @@
 # specific language governing permissions and limitations
 # under the License.
 
+ARG GO_VERSION
+FROM golang:${GO_VERSION} AS get-go
```

Comment:
Needs an `alpine-based` golang image since this Dockerfile uses an Alpine base image, since musl binaries are a different format